### ISSUE TITLE
Possibale to be based on jquery.sheet or handsontable

### ISSUE THREAD

**Issue by liuliming2008** [ethersheet-collective/EtherSheet#26]
TO avoid some UI work, like https://github.com/raphaelschroiff/NodeSpreadsheet
Only a sugguest without research

**Comment by JohnMcLear** [comment]
@liuliming2008 Maybe do some research and let us know if it's feasible?

**Comment by liuliming2008** [comment]
@JohnMcLear I have a research on these projects:
https://github.com/raphaelschroiff/NodeSpreadsheet: based on jquery.sheet
https://github.com/ethersheet-collective/EtherSheet 
https://github.com/vasconcelloslf/live-spreadsheet: based on  handsontable
https://github.com/AlexanderAA/jspreadsheet: based on based onhandsontable
https://www.ethercalc.org/4meevgwmab

Jquery.sheet and handsontable has almost all the basic functions for sheet, they only has UI.
https://www.ethercalc.org/4meevgwmab is a package of UI and server.
Jquery.sheet is more like a app than handsontable.

NodeSpreadsheet has online demo and can be setuped locally, based on jquery.sheet and sharejs.
Jquery.sheet  has a more complete UI wrapper than handsontable, but has more bugs and less community members, and not as developer freindly as handcontable. Although NodeSpreadsheet has a relatively full function package of a collaborative sheet.

Now here is two option with tech feasibility:
1 based on jquery.sheet and sharejs, like NodeSpreadsheet. Modify the UI tyle and Fix bugs of jquery.sheet.
2 based on handsontable and sharejs(or other OT framework),  build the whole UI wrapper for handsontable.

I prefer 2.

**Comment by liuliming2008** [comment]
Actually, https://github.com/raphaelschroiff/NodeSpreadsheet and https://www.ethercalc.org/4meevgwmab is basically usable, especially NodeSpreadsheet.

**Comment by JohnMcLear** [comment]
Nodespreadsheet last commit authored on 3 Dec 2012
Ethercalc is under a really bad license.

**Comment by liuliming2008** [comment]
Nodespreadsheet  is based on sharejs and jquery.sheet. Sharejs is active. Jquery.sheet is also active,  but not as popular and developer friendly as handsontable.

So I prefer 2:
2 based on handsontable and sharejs(or other OT framework), build the whole UI wrapper for handsontable.

**Comment by cooperq** [comment]
So we originally based the project on jquery.sheet but we found it too buggy when we tried to set up collaboration. It was too much of a hack to try to make every function of jquery.sheet go over the wire. So we wrote our own UI. The other advantage of this is that we were free to try to make a better spreadsheet UI and not just another one based off of the same old patterns (e.g. excel, libreoffice, visicalc). I think that ethersheet wiill have the most success in trying to subvert the existing spreadsheet paradigms. There are already a million other spreadsheets that work the same way, lets not be one million and one.
